### PR TITLE
Add Design Stickers Figma File to Design Files

### PR DESCRIPTION
**Pull request by GBKS**
Lists the newly created file for community sticker designs to the Design Files page in Resources. Hope people who want stickers can use this to easily print some, and that we also get some contributions to new stickers.

🧵[The preview](https://deploy-preview-873--bitcoin-design-site.netlify.app/guide/resources/design-files/).🪡

### CODE REVIEW

File: _compress_images_cache.yml [hunk 1]
```diff
@@ -1524,122 +1524,6 @@ assets/images/projects/projects@2x.jpg: projects@2x.jpg
...
-assets/animations/astronaut/images/img_0.png: img_0.png
-assets/animations/astronaut/images/img_1.png: img_1.png
-assets/animations/contribute/formatting/bitcoin-design-seal-miami-edition.png: bitcoin-design-seal-miami-edition.png
-assets/animations/contribute/formatting/bitcoin-design-seal.png: bitcoin-design-seal.png
-assets/animations/contribute/formatting/bitcoin-design-seal-miami-edition@2x.png: bitcoin-design-seal-miami-edition@2x.png
-assets/animations/contribute/formatting/bitcoin-design-seal@2x.png: bitcoin-design-seal@2x.png
-assets/images/home/banner/bits-and-pieces-fallback.png: bits-and-pieces-fallback.png
-assets/images/home/banner/bits-and-pieces-fallback@2x.png: bits-and-pieces-fallback@2x.png
-assets/images/guide/resources/design-files/bitcoin-wallet-backup-print-template.jpg: bitcoin-wallet-backup-print-template.jpg
-assets/images/guide/resources/design-files/bitcoin-wallet-backup-print-template@2x.jpg: bitcoin-wallet-backup-print-template@2x.jpg
```
Comment (sbddesign) [inline _compress_images_cache.yml:1642]: Same comment I made on #876. This is a lot of removed lines. Seems like an error.

**Review by sbddesign — CHANGES_REQUESTED** [review]
Looking good. Just flagged the issue with the compress cache.

File: _compress_images_cache.yml (hunk 1 above)
Comment (GBKS) [reply _compress_images_cache.yml:1642]: Double-checked, and this just removes duplicate lines. The script does it automatically for me. Not sure how these duplicate lines sneak in there, maybe when someone manually resolves conflicts?

**Review by sbddesign — APPROVED** [review]
🦞

**Review by sbddesign — DISMISSED** [review]

**Review by sbddesign — APPROVED** [review]

**Pull request merged**